sklearn/tree/tree.py

```diff
@@ -209,6 +210,20 @@ def fit(self, X, y, sample_weight=None, check_input=True,
         max_leaf_nodes = (-1 if self.max_leaf_nodes is None
                           else self.max_leaf_nodes)
 
+        if isinstance(self.min_samples_leaf, (numbers.Integral, np.integer)):
+            min_samples_leaf = self.min_samples_leaf
+        else:  # float
+            min_samples_leaf = int(np.ceil(self.min_samples_leaf *
+                                           n_samples))
```

Comment:
Beyond nitpicky and shouldn't matter here, but I generally used math functions directly when handling numbers.


    In [8]: %timeit np.ceil(1.5)
    The slowest run took 24.67 times longer than the fastest. This could mean that an intermediate result is being cached 
    1000000 loops, best of 3: 602 ns per loop
    In [7]: %timeit ceil(1.5)
    The slowest run took 22.57 times longer than the fastest. This could mean that an intermediate result is being cached 
    10000000 loops, best of 3: 98.8 ns per loop